src/components/EventAdd/stylesheet.scss

```diff
@@ -0,0 +1,124 @@
+@import "../../variables";
+
+.EventAdd {
+  display: flex;
+  flex-direction: column;
+  padding: 4px;
+
+  .add {
+    @include card;
+    background-color: $color-border;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+
+    table {
+      border-collapse: collapse;
+      table-layout: auto;
+      width: 100%;
+
+      tr:last-child {
+        border-bottom: none;
+      }
+
+      tr {
+        border-bottom: 1px solid $color-border;
+    
+        .label {
+          opacity: .5;
+          transition: opacity .2s;
+          font-weight: bold;
+          padding: 12px;
+          font-size: 13;
```

Comment:
Just a few very very minor changes to the styling to make the form a little more compact and consistent with other components:
```
padding: 10px;
font-size: .8em;
```